src/rdf4cpp/IRI.hpp

```diff
@@ -134,6 +134,48 @@ struct IRI : Node {
      * @return err if this is null, otherwise true iff this IRI is the default graph IRI
      */
     [[nodiscard]] TriBool is_default_graph() const noexcept;
+
+    /**
+     * Get the IRI for rdf:type
+     * @param node_storage optional custom node_storage where the returned IRI lives
+     * @return rdf:type IRI
+     */
+    [[nodiscard]] static IRI rdf_type(storage::DynNodeStoragePtr node_storage = storage::default_node_storage);
+
+    /**
+     * @return err if this is null, otherwise true iff this IRI is rdf:type
```

Comment:
add in the brief part something like `Check if this IRI is rdf:type`. The return type is correct but hard to parse.